attributes/packages.rb

```diff
@@ -4,12 +4,12 @@
 default['nodejs']['packages'] = value_for_platform_family(
   'debian' => node['nodejs']['install_repo'] ? ['nodejs'] : %w(nodejs npm nodejs-dev),
   %w(rhel fedora amazon) => node['nodejs']['install_repo'] ? %w(nodejs nodejs-devel) : %w(nodejs npm nodejs-dev),
-  'suse' => node['platform_version'].to_i < 42 ? %w(nodejs nodejs-devel) : %w(nodejs4 npm4 nodejs4-devel),
+  'suse' => node['platform_version'].to_i < 42 ? %w(nodejs10 npm10 nodejs10-devel) : %w(nodejs4 npm4 nodejs4-devel),
```

Comment:
We can probably nuke all the < 42 logic now that opensuse 42 is long EOL

Reply:
Yanked that from the implementation PR.